bio/gatk/baserecalibratorspark/environment.yaml

```
@@ -3,6 +3,6 @@ channels:
   - conda-forge
   - defaults
 dependencies:
-  - gatk4 ==4.1.4.1
+  - gatk4 ==4.2.0.0
   - openjdk =8
   - snakemake-wrapper-utils ==0.1.3
```

As above, I'd pin less stringently to allow for patches/bugfixes to be pulled in:
```suggestion
  - gatk4 =4.2
  - openjdk =8
  - snakemake-wrapper-utils =0.1.3
```